pillar.example

```diff
@@ -19,21 +18,24 @@ fail2ban:
         filter: sshd
         logpath: /var/log/auth.log
         maxretry: 6
-        port: ssh
+        port: ssh	
```

Comment:
There's trailing whitespace at the end of the line, which will fail the `yamllint`.  But since we've spent a while on this PR, I'll merge it and then fix this in the PR I have open after I rebase on this after merging.

Reply:
thx

Reply:
Fixed in #39.